config/crds/eventing_v1alpha1_knativeeventing_crd.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: knativeeventings.eventing.knative.dev
+spec:
+  group: eventing.knative.dev
+  names:
+    kind: KnativeEventing
+    listKind: KnativeEventingList
+    plural: knativeeventings
+    singular: knativeeventing
+    shortNames:
+      - ke
```

Comment:
does it need a short name? I am betting we do not want to use this very much.

Reply:
Based on my own experience in serving-operator, I always use command like 'k get ks -n knative-serving' to check the CR, so I think it can bring convenience in some degree.

Reply:
how about `kevent` which is similar as `ksvc`?